theme/js/components/dataset/search-result.vue

```diff
@@ -45,13 +45,68 @@
             {{ $filters.excerpt(description, 160) }}
           </p>
           <p class="fr-mb-0 text-mention-grey">
+            <Tooltip class="fr-hidden inline-sm">
+                <template #tooltip>
+                  <h5 class="fr-text--sm fr-my-0">{{$t("Metadata quality:")}}</h5>
+                  <QualityItem
+                    :passed="quality.dataset_description_quality"
+                    :messagePassed='$t("Data description filled")'
+                    :messageFailed='$t("Data description empty")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.license"
+                    :messagePassed='$t("License filled")'
+                    :messageFailed='$t("No license set")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.update_frequency && quality.update_fulfilled_in_time"
+                    :messagePassed='$t("Update frequency followed")'
+                    :messageFailed='quality.update_fulfilled_in_time ? $t("Update frequency not followed") : $t("Update frequency not set")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.has_open_format"
+                    :messagePassed='$t("File formats are open")'
+                    :messageFailed='$t("File formats are closed")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.temporal_coverage"
+                    :messagePassed='$t("Temporal coverage filled")'
+                    :messageFailed='$t("Temporal coverage not set")'
+                    class="fr-my-1w"
+                  />
+                  <QualityItem
+                    :passed="quality.spatial_coverage"
+                    :messagePassed='$t("Spatial coverage filled")'
+                    :messageFailed='$t("Spatial coverage not set")'
+                    class="fr-my-1w"
+                  />
+                  <div class="fr-grid-row fr-grid-row--right not-enlarged">
+                    <a
+                      href="https://guides.etalab.gouv.fr/qualite/documenter-les-donnees/"
```

Comment:
We should probably link to a dedicated *page*. @agarrone I think you made a draft but couldn't find it. Should we go for a *page*? Would we be ready to publish it when this PR goes live?

Reply:
Propositions to lighten-up the cards : 
- Allow 2 lignes of description only 
- Remove the underline on the score and the licence
- Let the date on its own line (and the score and licence on the same one)
- Hide the filters under an accordion by default 

Regarding your question @maudetes  I'm planning to publish a page on the quality guide. I'll try to draft it today :)

Reply:
See : https://github.com/etalab/guides.etalab.gouv.fr/pull/179